parquet-hadoop/pom.xml

```diff
@@ -76,7 +76,7 @@
     <dependency>
       <groupId>org.xerial.snappy</groupId>
       <artifactId>snappy-java</artifactId>
-      <version>1.1.2.6</version>
```

Comment:
could you please link the CVEs for snappy-java?

Reply:
There is no CVE that I found for snappy-java.  But here at Oracle the security team won't approve use of software more than 18 months or 5 versions old.  snappy-java 1.1.26 is 26 months and 9 releases old.  For that reason I included it in the PR.  (Same applies for fastutil and slf4j.)